packages/docs/src/routes/api/qwik/index.md

```diff
@@ -3679,7 +3679,7 @@ Options for the prefetch service worker.
 </tbody></table>
 **Returns:**
 
-[JSXNode](#jsxnode)&lt;'script'&gt;
+JSXNode&lt;'script'&gt;
```

Comment:
Please run `pnpm api.update` again, these get messed up by the build somehow :(

Reply:
fixed with #[26db654](https://github.com/QwikDev/qwik/pull/6824/commits/26db6545b56a337cedb4c97b2b496066515579c3)